port/unix/omrsysinfo.c

```diff
@@ -388,7 +389,12 @@ struct {
  */
 #define OMRPORT_SYSINFO_CGROUP_V1_AVAILABLE 0x1
 #define OMRPORT_SYSINFO_CGROUP_V2_AVAILABLE 0x2
-#define OMRPORT_SYSINFO_RUNNING_IN_CONTAINER 0x4
+
+/* Different states of PPG_processInContainerState. */
+#define OMRPORT_PROCESS_IN_CONTAINER_UNINITIALIZED 0x0
+#define OMRPORT_PROCESS_IN_CONTAINER_TRUE 0x1
+#define OMRPORT_PROCESS_IN_CONTAINER_FALSE 0x2
+#define OMRPORT_PROCESS_IN_CONTAINER_ERROR 0x4
```

Comment:
Why is this not `0x3` instead of `0x4`? These values are not combined anywhere that I can see that would necessitate them using different bits.